docs/installation.rst

```diff
@@ -0,0 +1,148 @@
+..
+      Copyright (C) 2021-2022 Modin authors
+
+      SPDX-License-Identifier: Apache-2.0
+
+============
+Installation
+============
+
+There are several ways to install unidist. Most users will want to install with
+``pip`` or using ``conda`` tool, but some users may want to build from the master branch
+on the `GitHub repo`_. The master branch has the most recent patches, but may be less
+stable than a release installed from ``pip`` or ``conda``.
+
+Installing with pip
+'''''''''''''''''''
+
+Stable version
+""""""""""""""
+
+unidist can be installed with ``pip`` on Linux, Windows and MacOS.
+To install the most recent stable release run the following:
+
+.. code-block:: bash
+
+  pip install unidist # Install unidist dependencies for Multiprocessing and sequential Python backends
+
+unidist can also be used with Dask, MPI or Ray execution backend.
+If you don't have Dask_, MPI_ or Ray_ installed, you will need to install unidist with one of the targets:
+
+.. code-block:: bash
+
+  pip install unidist[all] # Install unidist dependencies for all the backends
+  pip install unidist[dask] # Install unidist dependencies for Dask backend
+  pip install unidist[mpi] # Install unidist dependencies for MPI backend
+  pip install unidist[ray] # Install unidist dependencies for Multiprocessing and sequential Python backends
+
+unidist will automatically detect which execution backend you have installed and use that for
+scheduling computation!
+
+Release candidates
+""""""""""""""""""
+
+unidist follows `Semantic Versioning`_ and before some major or minor releases,
```

Comment:
Could you please explain more about pre-release builds. It is a part of general release processes but we decide if we add it to conda or not?

Reply:
We may want to upload pre-release builds on PyPI before some major or minor releases at some point in order to test and check if there are any problems. This is not related to conda because we first upload packages on PyPI.